PureInvoke/Functions/Invoke-AdvapiLookupAccountName.ps1

```diff
@@ -0,0 +1,71 @@
+
+function Invoke-AdvApiLookupAccountName
+{
+    <#
+    .SYNOPSIS
+    Calls the Advanced Windows 32 Base API (advapi32.dll) `LookupAccountName` function.
+
+    .DESCRIPTION
+    The `Invoke-AdvApiLookupAccountName` function calls the advapi32.dll API's `LookupAccountName` function, which looks up
+    an account name and returns its domain, SID, and use. Pass the account name to the `AccountName` parameter and the
+    system name to the `SystemName` parameter, which are passed to `LookupAccountName` as the `lpAccountName` and
+    `lpSystemName` arguments, respectively. The function returns an object with properties for each of the
+    `LookupAccountName` function's out parameters: `DomainName`, `Sid`, and `Use`.
+
+    .LINK
+    https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-lookupaccountnamea
+
+    .EXAMPLE
+    Invoke-AdvApiLookupAccountName -AccountName ([Environment]::UserName)
+
+    Demonstrates how to call this function by passing a username to the `AccountName` parameter.
+    #>
+    [CmdletBinding()]
+    param(
+        # The account name to lookup.
+        [Parameter(Mandatory)]
+        [String] $AccountName,
+
+        # The name of the system.
+        [String] $SystemName

```

Comment:
`$SystemName` should be `$ComputerName` for consistency with other functions in this module.